Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -47,13 +47,10 @@ class ConnectSetting: WithoutAccountSetting {
     
     override func onConfigureCell(_ cell: UITableViewCell) {
         super.onConfigureCell(cell)
-        
-        if AppConstants.MOZ_SHOW_FXA_AVATAR {
-            cell.imageView?.image = UIImage.templateImageNamed("FxA-Default")
-            cell.imageView?.tintColor = .lightGray
-            cell.imageView?.layer.cornerRadius = (cell.imageView?.frame.size.width)! / 2
-            cell.imageView?.layer.masksToBounds = true
-        }
+        cell.imageView?.image = UIImage.templateImageNamed("FxA-Default")
+        cell.imageView?.tintColor = .lightGray
+        cell.imageView?.layer.cornerRadius = (cell.imageView?.frame.size.width)! / 2
+        cell.imageView?.layer.masksToBounds = true
```

Comment:
would be nice to change this block to 
```
if let imageView = cell.imageView {

}```

Reply:
That was a sneak force unwrap. Didn't even notice it.